src/subscript/ri_wellmod/ri_wellmod.py

```diff
@@ -0,0 +1,586 @@
+):
+    """
+    Multiple inheritance used for argparse to get both
+    defaults and raw description formatter
+    """
+
+    # pylint: disable=unnecessary-pass
+    pass
+
+
+def get_resinsight_exe(version=DEFAULT_VERSION):
+    """
+    Return the path to a valid ResInsight install, False if not found.
+    """
+    ri_exe = shutil.which("ResInsight")
+    if ri_exe is not None:
+        return ri_exe
+
+    ri_exe = Path(RI_HOME + "/resinsight_" + version + "_RHEL7/ResInsight")
+    if ri_exe.exists():
+        return ri_exe
+
+    return False
+
+
+def get_parser():
+    """
+    Utility function to build the cmdline argument parser using argparse
+    (https://docs.python.org/3/library/argparse.html)
+    """
+
+    description = """
+  Script description.
+"""
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument("--verbose", "-vb", action="store_true", help="Verbose output")
+    parser.add_argument(
+        "--silent", "-s", action="store_true", help="Silence non-critical messages"
+    )
+    parser.add_argument("--debug", "-d", action="store_true", help="Debug mode")
+    parser.add_argument(
+        "well_project", help="Path to project with well paths and completions defined."
+    )
+    parser.add_argument("ecl_case", help="Path to initialized Eclipse case.")
+    parser.add_argument(
+        "--property_files",
+        nargs="*",
+        help="Additional input property files for PERM/NTG (GRDECL format)",
+    )
+    parser.add_argument(
+        "--output_file",
+        "-o",
+        default="well_defs.sch",
+        help="Output file (default=well_defs.sch)",
+    )
+    parser.add_argument(
+        "--tmpfolder",
+        "-tmp",
+        default="resinsight/ri_completions",
+        help="Output folder (default=tmp_ri_completions)",
+    )
+    parser.add_argument(
+        "--wells",
+        "-w",
+        nargs="+",
+        default=None,
+        help="Optional comma-separated list of wells (wildcards allowed) to generate completions \
+            for (default=all wells in project)",
+    )
+    parser.add_argument(
+        "--msw_wells",
+        "-msw",
+        nargs="+",
+        default=None,
+        help="Optional comma-separated list of wells (wildcards allowed) to generate msw \
+            well definitions for (default=none)",
+    )
+    parser.add_argument(
+        "--lgr",
+        "-l",
+        nargs="*",
+        help="Optional list of comma-separated LGR specs: WELLNAME:REF_I,REF_J,REF_K \
+          (wildcards allowed in well names)",
+    )
+    parser.add_argument(
+        "--lgr_output_file",
+        "-lo",
+        default="well_lgr_defs.inc",
+        help="Well LGR output file (default=well_lgr_defs.sch)",
+    )
+    parser.add_argument(
+        "--time_step",
+        "-t",
+        default=0,
+        help="Optional selection of time step to use for completion export (default=0)",
+    )
+    parser.add_argument(
```

Comment:
Please rename this option, `--version` should be reserved for the version of the script itself.